src/common/common_service/src/observer_manager.rs

```diff
@@ -232,9 +232,30 @@ impl NotificationClient for RpcNotificationClient {
     type Channel = Streaming<SubscribeResponse>;
 
     async fn subscribe(&self, subscribe_type: SubscribeType) -> Result<Self::Channel> {
-        self.meta_client
+        let mut res = self
+            .meta_client
             .subscribe(subscribe_type)
             .await
-            .map_err(RpcError::into)
+            .map_err(RpcError::into);
+
+        for retry in self.meta_client.get_retry_strategy() {
```

Comment:
Failover is already covered in `meta_rpc_client_method_impl`. So I guess this's not necessary?